src/composables/useRelatedFlawTrackers.ts

```diff
@@ -142,27 +143,41 @@ export function useRelatedFlawTrackers(
     }
   });
 
-  // Logic for isLoadingTrackers could be moved to useSingleFlawTrackers as an improvement
   watch(affectsBySelectedFlawId, (newRelatedAffects: Record<string, ZodAffectType[]>) => {
+    updateMultiFlawTrackers(newRelatedAffects, false);
+  }, { immediate: true });
+
+  watch(relatedFlaws, () => {
+    selectedRelatedFlaws.value.forEach((selectedFlaw, index) => {
+      const relatedIndex = relatedFlaws.value.findIndex(({ uuid }) => uuid === selectedFlaw.uuid);
+      if (index !== -1) {
```

Comment:
I think this was left behind after the renaming, `index` comes from the loop so it will never be `-1`
```suggestion
      if (relatedIndex !== -1) {
```

Reply:
Once this is fixed I'll leave my approval, rest LGTM
 ![](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExbHVlcXl6bmxnenJhYmJ3MGx4anA5N2dpdnNpOGt6M2g4MGZydDBrMCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/tIeCLkB8geYtW/giphy.gif)